Transition manuscript file to .qmd

eahowerton opened the pull request:
This is the original groundwork to transition from .Rmd to .qmd for the `hubEnsembles` manuscript. This includes the following changes: 

1. creating `hubEnsembles_manuscript.qmd` and removing `hubEnsembles_manuscript.Rmd`
2. updating the settings at the top of .qmd following `paper.qmd` that was generated by `rrtools`. This includes adding preliminary author information (related to #8). Note, this authorship information needs to be reviewed/updated, and I could not get the authors to show up in the .html file. 
3. changing the way figures/table are referenced and captions are defined to be compatible with Quatro

eahowerton commented:
I'll also note that rendering the `.qmd` file generated a new folder, `analysis/paper/hubEnsembles_manuscript_files` with a number of other files which I haven't pushed. Is this something I should include in the repo or should we add it to `.gitignore`?

lshandross reviewed (commented):
I think it looks pretty good, Emily. I am wondering if those files should be pushed as well since the HTML preview [here](https://htmlpreview.github.io/?https://github.com/Infectious-Disease-Modeling-Hubs/hubEnsemblesManuscript/blob/try-qmd/analysis/paper/hubEnsembles_manuscript.html) doesn't seem to show any of the figures (though it's possible that's just a quirk of a qmd-rendered HTML file). It would be nice if we could have a fully functional HTML preview

eahowerton commented:
Good point @lshandross - I just pushed the extra files. The HTML preview seems to be working on my end now

lshandross commented:
Mine too. It looks really good. Two more minor things though before I think it's ready to merge in:
1. Some of the math equations still don't render in the preview
2. A couple of the in-text references to figures have the word "figure" duplicated twice

eahowerton commented:
Good point @lshandross. I fixed the places where "figure" was duplicated, and cleaned up a few additional things. I couldn't find any equations that were not displaying, can you list the sections where I should look? Thanks!

lshandross commented:
The equations under the Mathematical definitions and properties of ensemble methods section don't show up for me in the preview (see below), though it's fine when I open the file locally
![Screenshot 2024-02-16 093327](https://github.com/Infectious-Disease-Modeling-Hubs/hubEnsemblesManuscript/assets/57642277/e24347b6-cb1b-4ac1-9bda-bb1db15c8ea5)

eahowerton commented:
Noting it seems the problem with preview is no longer an issue.

annakrystalli commented:
Hello @eahowerton and @lshandross , I started to review and make corrections and decided to just go ahead and commit directly to the PR instead of opening another PR. Hope you don't mind (I thought it would be more efficient!)

The main things I fixed:
- A couple of table cross-references that weren't working
- Corrected the authors syntax to conform to Quarto expectations
- Made code reproducible when running interactively by using `here()`
- Styled the paper and code files in R/ for better readability (It's a good idea to run`styler::tidyverse_style()` regularly when making changes to the code in the paper. Must be in the Source editor though, doesn't work in the Visual editor)
- Added a draft README in the style of `rrtools` which is much more relevant to a research compendium.  Once we have DOIs and publication details for papers/pre-prints etc we can add them. We also need to complete the uthors field in each citations but I've left that for now as wasn't sure what format you wanted. (#26 )

Another cool thing included in the README is a button to launch a binder in the cloud where theoretically anyone can interact with the code. It's not currently working and needs a bit of work. In any case, I've opened an issue (#25 ) to revisit it when I have some time.

annakrystalli reviewed (approved):
LGTM

Added some commits and comments in PR
